demo/Nonlinear Virus Spread.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p>From a user perspective, it might not be clear that we are using a <code>Delta</code> node as we are hiding it in the model specification language</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@Chengfeng-Jia